### ISSUE TITLE
Retreat RSVP List

### ISSUE THREAD

**Issue by clone1018** [gratipay/inside.gratipay.com#392]
## What?

The purpose of the Gratipay company retreat is to strategize about the year ahead, sprint on Gratipay and related projects, and to continue to build meaningful relationships.

## What's the Goal?
To be determined (#393).

## When?

Starts: Friday January 1st, 2016 at 5:00pm
Ends: Monday, January 4th, 2016 at 12:00pm

## Where?

716 Park Road, Ambridge, PA 15003

### Yes: 3
1 @whit537
1 @kaguillera 
1 @timothyfcook 

### Maybe: 3
1 Miguel
1 @kzisme
1 @jritorto 
1 @dmk246

**Comment by clone1018** [comment]
Alright @gratipay let's do this!

**Comment by kzisme** [comment]
I am currently a maybe until the date gets closer :)

**Comment by techtonik** [comment]
I found that I may try to apply for a grant to visit you. But on December 30, 2015 I will likely to be here - at 32C3 - https://events.ccc.de/2015/09/14/32c3-call-for-participation-en/

**Comment by whit537** [comment]
+1!

**Comment by whit537** [comment]
@mattbk @webmaven Any chance we'll see you at the retreat? :-)

**Comment by whit537** [comment]
@kaguillera @timothyfcook Retreat? :-)

**Comment by whit537** [comment]
@rummik @jritorto Gonna make it to the Gratipay retreat? :-)

**Comment by timothyfcook** [comment]
I'm a tentative "Yes" for part of the event.

**Comment by whit537** [comment]
@timothyfcook Yay! I've added you under "Maybe" in the ticket description. :-)

**Comment by mattbk** [comment]
I cannot be there, it's a little too far from home.

**Comment by webmaven** [comment]
@whit537:
> Any chance we'll see you at the retreat? :-)

None, I'm afraid (unless transportation and lodging could be covered somehow, and even then it wouldn't be certain). :disappointed:

**Comment by whit537** [comment]
@mattbk You've been pretty heavily involved since the Gratipocalypse. I'd be awesome to have you here. Is cost the issue? Tickets from Minneapolis (yes?) to Pittsburgh are $350. Can we talk about using Gratipay funds to help get you here?

Ditto for @rohitpaulk and @rorepo. :-)

**Comment by whit537** [comment]
> (unless transportation and lodging could be covered somehow, and even then it wouldn't be certain)

@webmaven I'd be open to talking about making some funds available. :-)

**Comment by whit537** [comment]
@webmaven et al. To be clear: lodging is covered. The past two years, we've been able to accommodate everyone comfortably at my house. I don't anticipate *more* people this year. :-)

**Comment by whit537** [comment]
Also, the airport is ~20 minutes away, so it's really easy for us to arrange rides to and fro. Last year we paid for most food out of Gratipay operations as well. We went out a few times (depends on how long you stay for), and people were generally on their own then.

Basically, if we can get you here, then it should be pretty cheap once you're here.

**Comment by clone1018** [comment]
Yeah, from what I've experienced the actual retreat is cheap. Me and Miguel will make some meals for everyone for those days, there's coffee and teas. You usually don't leave the house at all for the weekend haha.

**Comment by whit537** [comment]
> You usually don't leave the house at all for the weekend haha.

And for a few days after, if there's a polar vortex! :snowman: :dancer:

**Comment by mattbk** [comment]
I'd be flying from an eastern ND airport, but I can't take the time off from work and other responsibilities.

**Comment by rummik** [comment]
@whit537 The startup I'm at is going through [Alpha Lab](http://alphalab.org/), and with the crazy deadlines we've set for ourselves I'm not sure I'll be able to make it this time >.>;

**Comment by whit537** [comment]
@rummik Nice! Good luck, and if you need to take a break, you know where/when to find us! :-)

@mattbk Okay, fair enough. We'll miss you. :-)

**Comment by jritorto** [comment]
I'd like to show up here and there for some coding.

**Comment by whit537** [comment]
@jritorto Huzzah! :dancer:

**Comment by whit537** [comment]
@jritorto I'll put you down as a maybe overall?

**Comment by jritorto** [comment]
yes please!

**Comment by whit537** [comment]
Cool, done. :-]

**Comment by webmaven** [comment]
@whit537:
> I'd be open to talking about making some funds available.

Chad, I'm looking at almost $500 in airfare between ABQ <-> PIT close to the dates of the retreat. I'm pretty strapped for funds right now. How much could be covered by GP?

**Comment by whit537** [comment]
@webmaven Good question! We've got $1,947.30 in operations at PNC right now. We bring in about $200 per week and we spend about $100 per week. If we drop below $1,500 we get hit with a $12 service fee.

Other expenses I have my eye on:

 - OuiShare (#314)
 - lawyer for #242
 - retreat food and incidentals

Too bad http://cobudget.co/ doesn't really exist yet. :-)

What else do folks want to spend money on? What should our travel budget be?

**Comment by whit537** [comment]
P.S. Hey @dmk246 @oakes, wanna come to the Gratipay Retreat? :-)

**Comment by whit537** [comment]
I AM EXCITED ABOUT THE GRATIPAY COMPANY RETREAT. MIGUEL, LUKE, AND I ARE GOING TO HAVE A FANTASTIC TIME. NOBODY ELSE WILL BE THERE.

:turtle: :cocktail: :roller_coaster:

**Comment by webmaven** [comment]
I think at this point you can mark me as not coming, unfortunately.

**Comment by whit537** [comment]
@webmaven Okay. We may be able to video conference people in, as well.

**Comment by clone1018** [comment]
Sucks that so many people aren't going to be able to make it this year. :(

**Comment by whit537** [comment]
@kzisme Now that you made it through finals ... how does the weekend of Jan 1-3 look for you? I'd *really* love to see you make the retreat! :dancer:

**Comment by whit537** [comment]
You and Miguel still in, @clone1018?

**Comment by whit537** [comment]
I believe @kaguillera is planning on being there Saturday (yes?).

@timothyfcook What are your plans? Want to travel w/ @kaguillera on Saturday?

**Comment by whit537** [comment]
I just had a phone call with @timothyfcook. I'm moving him to a "yes" on this ticket. He's in for a half-day on Saturday.

**Comment by whit537** [comment]
I just called and left a message (I think?) for @clone1018.

**Comment by whit537** [comment]
[Looks like](https://botbot.me/freenode/gratipay/2014-11-12/?msg=25371195&page=4) I was worried last year, too:

> clone1018: Are you still coming to my house in January?
clone1018: I don’t care if you’re the only one, we’re still having the Gratipay retreat.
If you won’t be here then I’ll have the Gratipay retreat by myself. :-)

Worked out [great](https://medium.com/gratipay-blog/at-our-second-retreat-we-met-our-third-users-5b473a2d1464) last year, I'm sure we'll be fine again this year. :-)

**Comment by kzisme** [comment]
I don't think I can make it on the 1st but another day I may be able to attend if that is allowed

**Comment by whit537** [comment]
@kzisme Certainly allowed. How early can you get here on Saturday (the 2nd)? We were planning to start at 9am. You can also get here late Friday and we'll have a bed for you. 716 Park Road 15003.

**Comment by jritorto** [comment]
Getting fired up for the vintage programming immersion idea!  Maybe we
should come up with a kids version, too?

**Comment by kaguillera** [comment]
Yes I will be there on Saturday 2nd January. I will try to make it for 9 am it depends on the buses

**Comment by clone1018** [comment]
Sorry for the delayed response @whit537 

It looks like I'll be unable to attend this year unfortunately. My wife had a pretty expensive surgery two weeks ago and it looks like I wont be able to afford it this year. I'm assuming that means Miguel will be dropping out too, but I'll check in with him.

**Comment by whit537** [comment]
Oh no! Sorry about your wife's surgery. :-(

Should we talk about tapping into Gratipay funds?

**Comment by clone1018** [comment]
Well looking at it from a business perspective it doesn't really make sense. I'm not an "active contributor" for the most part to Gratipay, according to https://github.com/gratipay/gratipay.com/pulse/monthly

Ideally Gratipay funds would go towards the more productive members (which I know you've already tried haha).

**Comment by whit537** [comment]
> Should we talk about tapping into Gratipay funds?

[Looks like](http://www.kayak.com/flights/DFW-PIT/2015-12-31/2016-01-04) it's still only $270 for a flight for the weekend or [a bit less](http://www.kayak.com/flights/DFW-PIT/2015-12-31/2016-01-09) if you stay longer. You're not contributing a lot of code right now, but it's cheaper to get you here than, say, @rohitpaulk. You've been around a long time, and you're important from a morale and strategy point of view. 

I hate to put the pressure on, but the retreat is going to look significantly different without you here. Even with you here it'll be slimmer than years past, but without you here I don't think it'll make sense to have any kind of (videotaped) formal agenda. I'm fine with that if that's what makes sense this year, but in my view it's worth spending $300 to pull off a retreat that keeps the Gratipay flame alive for one more year.

If it's about time away from your wife in addition to financial constraints then that's another matter. If we can't pull off a formal retreat this year then that's just the reality of where we're at right now and I can accept that.

**Comment by whit537** [comment]
May I buy you a flight, @clone1018? :-) What dates/times?

**Comment by whit537** [comment]
Alright, @clone1018. You're off the hook. :fishing_pole_and_fish:

I just called Dan and called off Saturday. We'll do something informal with whomever shows up.

:fire: :chart_with_downwards_trend: :fried_shrimp: 

We didn't have a retreat the first year, in 2013. If we can avoid additional crises and apocalypses in 2016, maybe we can come back together in 2017 and get back on track.

!m *

!m @clone1018. Take care of your wife!

**Comment by whit537** [comment]
P.S. Just got off the phone1018 with @clone1018. Hard to get this across on GitHub (not enough [emotional bandwidth](http://www.kapor.com/writing/emotional-bandwidth/)), a little easier over the phone—I'm pretty sure we're all okay (I hope?). No blame, just the reality of our situation this year, and it happened to really come down to the wire. Gratipay is in some lean times, no doubt about that. Let's call it an almost-retreat this year, and regroup towards 2017.

**Comment by whit537** [comment]
!m *

**Comment by clone1018** [comment]
!m *

**Comment by whit537** [comment]
Alright, the party for tomorrow night is coming together swimmingly, but it looks like that's going to be the extent of the retreat this year. Everyone have a happy new year. Let's see what happens and whether we get together again in 2017! :-)

**Comment by whit537** [comment]
> it looks like that's going to be the extent of the retreat this year.

Looks were deceiving! @mattbk [popped up](https://github.com/gratipay/inside.gratipay.com/issues/774#issuecomment-240292897) like a serendipitous Pokémon in Pittsburgh this week, for a DOE conference, while @oakes is also in town for Abstractions (#534). The three of us pulled together a meetup last night on short notice (@kaguillera was unable to attend), which by the end of the night we were ready to declare the 2016 Gratipay Company Retreat! 💃

The evening started with @mattbk and I punching a penny on the roof of a parking garage in Station Square ("How about 8:53 on the roof of the Station Square parking garage?" "Okay. That's pretty sketchy though." The full moon lent atmosphere; we barely dodged security) before joining @oakes and walking way too far down to the South Side. @Gratipay bought the first round, at [Acacia](http://acaciacocktails.com/), followed by a pint at [Piper's Pub](http://www.piperspub.com/). Here's a terrible, terrible picture, hastily snapped by @oakes towards the end of the night, while waiting for an Uber:

![img_20160818_240113298](https://cloud.githubusercontent.com/assets/134455/17771370/9410d910-6510-11e6-9182-637d1eb4f6c7.jpg)

*l. to r.: @mattbk @whit537 @oakes*

We covered a lot of ground, both walking and in conversation. Probably most directly helpful, @mattbk and I were able to go over #432, discussing it in light of the incremental approach to transformational change exhibited in the Center for Open Science's strategic plan (#755).

So there. We did it after all! One implication is that we may have the option of scheduling next year's retreat for August instead of January, if we want to. ;-)

!m * 💃
